src/pytest_inmanta_lsm/plugin.py

```diff
@@ -179,21 +196,6 @@ def remote_orchestrator_host(
         else (str(remote_orchestrator_container.orchestrator_ips[0]), remote_orchestrator_container.orchestrator_port)
     )
 
-    for _ in range(0, 10):
-        try:
-            http = "https" if inm_lsm_ssl.resolve(request.config) else "http"
-            response = requests.get(f"{http}://{host}:{port}/api/v1/serverstatus", timeout=2, verify=False)
-            response.raise_for_status()
-        except Exception as exc:
-            LOGGER.warning(str(exc))
-            time.sleep(1)
-            continue
-
-        if response.status_code == 200:
-            return host, port
-
-    raise RuntimeError(f"Couldn't reach the orchestrator at {host}:{port}")
```

Comment:
I moved this out of this fixture and into the `remote_orchestrator_access` one to be able to use the `RemoteOrchestrator.session` object.  This changes the behavior of the fixture, but not for the expected usage of the tool.

I also changed what is being done in the `RemoteOrchestrator` (environment configuration is now done in the fixture), which is another behavioral change, which I also think is okay as it doesn't change the behavior for the expected usage of the tool.